### PR TITLE
Autoformat scrabble.js

### PR DESCRIPTION
**Pull request by Avamander**

### CODE REVIEW

File: client/javascript/scrabble.js [hunk 1]
```diff
@@ -31,440 +29,398 @@ if (typeof exports == 'object') {
...
-    var string =  'Square type ' + this.type + ' x: ' + this.x;
+    var string = 'Square type ' + this.type + ' x: ' + this.x;
     if (this.y != -1) {
         string += '/' + this.y;
     }
     if (this.tile) {
         string += ' => ' + this.tile;
-        if (this.tileLocked) {
-            string += ' (Locked)';
+        if (this.tileLocked) {string += ' (Locked)';
```
Comment (hanshuebner) [inline client/javascript/scrabble.js]: Sorry, but no.

**Comment by Avamander** [comment]
Fixed that error.

File: client/javascript/scrabble.js [hunk 2]
```diff
@@ -475,75 +431,62 @@ Board.fromServerData = function(data) {
...
-                        function(square) {
-                            if (square.tile) {
-                                if (square.tile.isBlank() && !blankSquare) {
-                                    blankSquare = square;
-                                } else if (square.tile.letter == letter) {
-                                    return true;
-                                }
-                            }
-                        });
+        function(square) {
```
Comment (hanshuebner) [inline client/javascript/scrabble.js]: I disagree with this indentation.

File: client/javascript/scrabble.js [hunk 3]
```diff
@@ -553,39 +496,28 @@ Rack.prototype.findLetterSquare = function(letter, includingBlank) {
...
     }
     for (var i = 0; i < letterDistribution.length; ++i) {
-	var letterDefinition = letterDistribution[i];
-	
-	var tile = new Tile(letterDefinition.letter || " ", letterDefinition.score);
-        if (letterDefinition.letter) {
-            letterBag.legalLetters += letterDefinition.letter;
+        var letterDefinition = letterDistribution[i];
+        var tile = new Tile(letterDefinition.letter || " ", letterDefinition.score);
+        if (letterDefinition.letter) {letterBag.legalLetters += letterDefinition.letter;
```
Comment (hanshuebner) [inline client/javascript/scrabble.js]: Sorry, but your autoformatter fails.  I don't trust it enough.

**Comment by Avamander** [comment]
Didn't expect the formatter to be flawless. Better?

**Comment by hanshuebner** [comment]
As I wrote, I reserve the right to refuse formatting changes, and I'm
refusing these.  Too little gain, too many issues.

2017-04-19 19:42 GMT+02:00 Avamander <notifications@github.com>:

**Comment by hanshuebner** [comment]
Sorry.

**Comment by Avamander** [comment]
Understandable, though it really needs fixing, it's rather awful at the moment.

**Pull request closed without merging**